tooling/test-utils/src/types.ts

```
@@ -0,0 +1,4 @@
+export type HTTPMethod = "GET" | "POST" | "PUT" | "DELETE" | "PATCH" | "HEAD" | "OPTIONS";
+// eslint-disable-next-line ts/no-empty-object-type
+export interface EmptyObject {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)